client/src/pages/EnsemblesPage.tsx

```diff
@@ -1,60 +1,101 @@
 import TextHeadline from "../components/atoms/TextHeadline";
 import { useFetch } from "../hooks/use-fetch";
 import { useStore } from "../store/useStore";
-import { useEffect } from "react";
-import { EnsembleType } from "../types/EnsembleType";
+import { useEffect, useState } from "react";
 import TextBody from "../components/atoms/TextBody";
 import { Dropdown } from "../components/molecules/Dropdown";
 import { InstrumentType } from "../types/InstrumentType";
 import Button from "../components/atoms/Button";
 import { ICON_NAMES } from "../components/atoms/Icon/IconNames";
-import EnsembleCard from "../components/molecules/EnsembleCard";
+import PostCard from "../components/molecules/PostCard";
+// import { EnsemblePostsType } from "../types/EnsemblePostsType";
+// import { EnsembleType } from "../types/EnsembleType";
+import { PostWithEnsembleType } from "../types/PostWithEnsembleType";
 
 //TODO: this page will be ensembles posts page and will need to be reworked.
```

Comment:
Is this TODO done? do we need to change the name of the file ???

Reply:
I wasn't sure, but in my opinion as there is only one type of post and we don't just have a collection of ensembles its okay to name it like how it is now

Reply:
It was a WIP when I handed it over. This page has now the posts.. We could rename it to avoid confusion